src/domain/innovation-space/branding/branding.interface.ts

```diff
@@ -1,14 +1,8 @@
 import { BaseAlkemioEntity } from '@domain/common/entity/base-entity';
-import { IVisual } from '@domain/common/visual';
 import { Field, ObjectType } from '@nestjs/graphql';
 
 @ObjectType('Branding')
 export class IBranding extends BaseAlkemioEntity {
-  @Field(() => IVisual, {
```

Comment:
Removing this is not part of the scope of this PR and requires further discussion.